workers/unity/Assets/Playground/Scripts/Worker/WorkerConnectorBase.cs

```diff
@@ -54,6 +54,11 @@ public async Task Connect(string workerType, ILogDispatcher logger)
                         new LogEvent("Cancelling connection attempt")
                             .WithField("WorkerType", workerType)
                             .WithField("Reason", "Required worker failed to connect"));
+#if UNITY_EDITOR
+                    logger.HandleLog(LogType.Warning,
+                        new LogEvent("A local runtime may not have been started. If not, you can start one from 'SpaitalOS -> Local launch' or by pressing Ctrl-L")                      
```

Comment:
I'd reword this as something you have to check unless we are checking that this is the likely reason. If I read this I'd think it had failed to detect a runtime, not that it was just a guess.

Reply:
Will do